docs/source/en/tutorials/inference_with_big_models.md

```diff
@@ -0,0 +1,71 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Working with big models
+
+A modern diffusion model ([SDXL](../using-diffusers/sdxl.md), for example) is not just a single model but a collection of multiple models. SDXL has four different model-level components:
+
+* A variational autoencoder (VAE)
+* Two text encoders
+* A denoiser (which has a UNet architecture)
+
+Usually, the text encoders and the denoiser are bigger and much larger in size compared to the VAE. 
+
+As models keep getting bigger and better, it’s possible your model is so big that even a single copy won’t fit in RAM. That doesn’t mean it can’t be loaded: if you have one or several GPUs, this is more memory available to store your model. In this case, it’s better if your model checkpoint is split into several smaller files that we call checkpoint shards.
+
+When a text encoder checkpoint has multiple shards ([T5-xxl for SD3](https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers/tree/main/text_encoder_3), for example), it will be automatically handled by the `transformers` library as it is a required dependency of Diffusers when using the [`StableDiffusion3Pipeline`]. 
```

Comment:
I would clarify what specifically is meant by "automatically handled by the Transformers library" and consider whether it is even worth mentioning this little detail to users (ie, why is it important for them to know that it is handled by Transformers?).

```suggestion
When a text encoder checkpoint has multiple shards, like [T5-xxl for SD3](https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers/tree/main/text_encoder_3), it is automatically handled by the [Transformers](https://huggingface.co/docs/transformers/index) library as it is a required dependency of Diffusers when using the [`StableDiffusion3Pipeline`]. 
```